pandas/_libs/tslibs/timezones.pyx

```diff
@@ -27,14 +27,10 @@ from dateutil.tz import (
     tzlocal as _dateutil_tzlocal,
     tzutc as _dateutil_tzutc,
 )
+import numpy as np
 import pytz
 from pytz.tzinfo import BaseTzInfo as _pytz_BaseTzInfo
 
-UTC = pytz.utc
```

Comment:
IIRC the idea was to have a a constant here `UTC = the default utc object` that could be used everywhere where we need a UTC constant. Did you find that we weren't consistent about using this?

Reply:
> Did you find that we weren't consistent about using this?

i think thats accurate